controllers/front/notify.php

```diff
@@ -55,9 +55,11 @@ public function postProcess()
             $saferPayOrderId = $orderRepo->getIdByOrderId($orderId);
 
             $saferPayOrder = new SaferPayOrder($saferPayOrderId);
-            $order = new Order($orderId);
             $assertResponseBody = $this->assertTransaction($cartId);
```

Comment:
reference for the future - when there's object modified in multiple places it would be the best to pass it down the three. Maybe even to adapt such design pattern as Pipeline or something. https://medium.com/@bonnotguillaume/software-architecture-the-pipeline-design-pattern-from-zero-to-hero-b5c43d8a4e60